### PR TITLE
Fix MTU config with TAP Script

### PR DESCRIPTION
**Pull request by dpathrabe**
Signed-off-by: dpathrabe <deepak.p.pathrabe@intel.com>

### CODE REVIEW

**Comment by ipdksw** [comment]
Result:FAILURE JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-1/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/dd3d5c8bc94d7adca801815f2f1ef0d382d9175f

**Comment by dpathrabe** [comment]
[mtu_config_tap_fix_exec.log](https://github.com/ipdk-io/ovs/files/9498457/mtu_config_tap_fix_exec.log)
[mtu_config_tap_fix_build.log](https://github.com/ipdk-io/ovs/files/9498459/mtu_config_tap_fix_build.log)

Attaching build and execution logs 

Fix comment : Adding OSError exception to send_packet function call, which is an assertion in this scenario to test if packet sending is failed on account of MTU and packet size difference

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-2/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/dd3d5c8bc94d7adca801815f2f1ef0d382d9175f

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-3/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/92783a0fe4181f40e74c87a7e939abd86264b385

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-4/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/0d31ffbcdff03a1416bfa78c6a48a9a50e1208d2

**Comment by ipdksw** [comment]
Result:ABORTED JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-5/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/0d31ffbcdff03a1416bfa78c6a48a9a50e1208d2

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-6/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/0d31ffbcdff03a1416bfa78c6a48a9a50e1208d2

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-7/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/0d31ffbcdff03a1416bfa78c6a48a9a50e1208d2

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-8/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/0d31ffbcdff03a1416bfa78c6a48a9a50e1208d2

File: ptf_tests/tests/mtu_config_test_with_tap.py [hunk 1]
```diff
@@ -101,23 +101,33 @@ def runTest(self):
...
         try:
-            verify_packet(self, pkt, port_id=port_ids[1][1], timeout=None)
+           send_packet(self, port_ids[self.config_data['traffic']['send_port'][0]][1], pkt)
         except Exception as err:
-            outpcap = tcpdump_get_pcap(port_list[1])
-            mtudiffstr = "truncated-ip " + str(mtudiff) + " bytes missing!"
-            result =  tcdump_match_str(outpcap, mtudiffstr)
-            if result == True :
-                print(f"PASS: Assertion verified as per difference in MTU and packet size")
+           errstr = f"Message too long"
```
Comment (dkaranjai) [inline ptf_tests/tests/mtu_config_test_with_tap.py:92]: can remove the f

**Review by dkaranjai — CHANGES_REQUESTED** [review]

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-9/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/f6b019434d4303b6ebe2b9af1580a3bd5dc6bd42

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-10/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/f6b019434d4303b6ebe2b9af1580a3bd5dc6bd42

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-11/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/c7ad3ef530be619691a4ab2d0108ec8e08de8d16

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-12/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/c7ad3ef530be619691a4ab2d0108ec8e08de8d16

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-13/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/c7ad3ef530be619691a4ab2d0108ec8e08de8d16

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-14/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/4f1bc06d1c25750cec49cbbcb3096bc92b91d3e7

**Review by sijubaker — APPROVED** [review]

**Comment by dpathrabe** [comment]
required changes done

**Comment by ipdksw** [comment]
Result:ABORTED JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-15/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/dd3d5c8bc94d7adca801815f2f1ef0d382d9175f

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-63-16/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/dd3d5c8bc94d7adca801815f2f1ef0d382d9175f

**Comment by dpathrabe** [comment]
Closing this PR and created a new PR #71

**Comment by dpathrabe** [comment]
Deleting branch

**Pull request closed without merging**